en/lessons/parsers/introduction.md

```diff
@@ -0,0 +1,62 @@
+---
+version: 1.0.0
+title: Introduction to Parser Combinators
+---
+
+{% include toc.html %}
+
+Parsers are ubiquitous in Haskell litterature because, thanks to Parser Combinators it illustrates the power of composition.
```

Comment:
```suggestion
Parsers are ubiquitous in educational Haskell literature because parser combinators can be used to illustrate the power of composition.
```

Correct me if I'm wrong about the implication here being that parser combinators are a useful tool in teaching Haskell. 

I have replaced the pronouns to make the sentence a little clearer (in general however, "they" is used to replace a plural subject).